clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -28,18 +33,22 @@ class ApiClient(apiUrl: String, accessKey: String, secretKey: String) {
   private val metadataApi = new api.MetadataApi(client)
   private val branchesApi = new api.BranchesApi(client)
 
-  private val storageNamespaceCache = CacheBuilder.newBuilder().expireAfterWrite(2, TimeUnit.MINUTES).build[String, String]()
+  private val storageNamespaceCache =
+    CacheBuilder.newBuilder().expireAfterWrite(2, TimeUnit.MINUTES).build[String, String]()
```

Comment:
It would've done it in case `CacheBuilder.newBuilder().expireAfterWrite(2, TimeUnit.MINUTES).build[String, String]()` was longer than 100 (including the spaces before)